sos/report/plugins/ostree.py

```diff
@@ -0,0 +1,30 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+
+
+class OSTree(Plugin, IndependentPlugin):
+
+    short_desc = 'OSTree'
+
+    plugin_name = 'ostree'
+    profiles = ('system', 'sysmgmt', 'packagemanager')
+    files = ('/ostree',)
+
+    def setup(self):
+        self.add_copy_spec("/ostree/repo/config")
```

Comment:
Just an ostree-newbie question: Cant be there some credentials to obfuscate, in a repo config?

Reply:
Looks like a no from the man page.. but I'm very new to it too.